components/CropSelector/CropSelector.vue

```diff
@@ -97,9 +99,28 @@ export default {
        */
       this.$emit('valid', event);
     },
-    handleAddClicked() {
+    async handleAddClicked(newCrop) {
```

Comment:
Could everything in this method be in the if(newCrop) conditional? Seems like if no crop was created then there is no need to clear the cache and reload the map.

If so, this will apply to all of the other parent components also.

Reply:
I did it that way as a from of error proofing. Currently, ‘add-clicked’ is emitted with the ‘newCrop’ payload only if the ‘SelectorBase’ found the confirmation message and got the name of the new crop (this also closes the popup). If ‘SelectorBase’ cannot find the ‘newCrop’ (add page structure changes or some other reason) and the user closes the popup manually, currently it will still clear the cache and reload the map. Let me know if that’s something that should be implemented or not.

Reply:
It seems like getting `null` back indicates that no new crop would have been created and thus, there is no reason to clear and refresh the cache with a new API call.